core/src/components/checkbox/checkbox.scss

```diff
@@ -50,30 +60,90 @@ label {
   opacity: 0;
 }
 
+.checkbox-wrapper {
+  display: flex;
+
+  flex-grow: 1;
+
+  align-items: center;
+
+  height: 100%;
+  
+  cursor: inherit;
+}
+
+/**
+ * Allows us to set the background of the container
+ * While keeping the color and opacity separate.
+ */
+.checkbox-wrapper::after {
+  @include button-state();
+    
+  z-index: -1;
```

Comment:
I'm not sure I understand the comment. Can you clarify?

Reply:
Ah whoops, this isn't actually needed anymore. It was a leftover to enable the new background CSS properties, but we've removed those. Thanks for the callout!